src/compat.h

```diff
@@ -443,6 +545,31 @@ get_attname_compat(Oid relid, AttrNumber attnum, bool missing_ok)
 				 NULL)
 #endif
 
+/* index_get_next
+ *
+ * index_get no longer exists as of c2fe139. As tables are now generalized to
+ * cover non-heap-based things, it is incorrect to return a HeapTuple in the
+ * general case. To allow us to migrate to the new APIs more incrementally, we
+ * provide our own, temporary, implementation of index_get_next, which should be
+ * removed at some point when all our usages are replaced with index_getnext_slot
+ */
+// #if PG12
```

Comment:
What's this commented code? Also C++ style comments.

Reply:
These are TODOs, I'm not sure yet if the code is right. It's deliberately mis-styled as preventions from going in.